molgenis-data-vcf/src/main/java/org/molgenis/data/vcf/VcfReaderFactoryImpl.java

```diff
@@ -41,6 +44,13 @@ public VcfReader get()
 			if (file.getName().endsWith(".gz"))
 			{
 				inputStream = new GZIPInputStream(inputStream);
+			} 
+			else if (file.getName().endsWith(".zip"))
+			{
+				   ZipFile zipFile = new ZipFile(file.getPath());
+				   Enumeration<? extends ZipEntry> e = zipFile.entries();
+				   ZipEntry entry = (ZipEntry) e.nextElement(); // your only file
```

Comment:
Some OS-es add system-specific entries to a zipfile when you create them from the finder / explorer.
Could you make it so that it imports the first entry with extension .vcf and explains why it fails if no such entry is found?